src/components/ProgressIndicator/ProgressIndicator.js

```diff
@@ -88,21 +96,35 @@ export const ProgressStep = ({ ...props }) => {
 
   return (
     <li className={classes}>
-      {currentSvg || completeSvg || incompleteSvg}
-      <ProgressStepLabel className={`${prefix}--progress-label`}>
-        {label}
-      </ProgressStepLabel>
-      {componentsX &&
-      secondaryLabel !== null &&
-      secondaryLabel !== undefined ? (
-        <p className={`${prefix}--progress-optional`}>{secondaryLabel}</p>
-      ) : null}
-      <span className={`${prefix}--progress-line`} />
+      <div
+        className={classnames(`${prefix}--progress-step-button`, {
+          [`${prefix}--progress-step-button--unclickable`]: !onClick || current,
+        })}
+        role="button"
+        tabIndex={!current && onClick ? 0 : -1}
+        onClick={!current ? onClick : undefined}
```

Comment:
the PR description states that each progress step will have a click handler attached, but it looks like the current progress step will not be tabbable or receive the click handler. I just want to confirm, is this the expected behavior?

Reply:
Yeah @emyarod in our user testing we got feedback that we shouldn't allow them to click on the current step.

Reply:
gotcha, thanks for clarifying @scottdickerson !